testing/package_testing.py

```diff
@@ -33,7 +37,7 @@ def has_pytest_files(recipe_dir):
 def create_test_env(pkg_name, prefix):
     # cmd = ['$MAMBA_EXE' ,'create','--prefix', prefix,'--platform=emscripten-32'] + [pkg_name] #+ ['--dryrun']
     cmd = [
-        f"""$MAMBA_EXE create --yes --prefix {prefix} --platform=emscripten-32   python "pyjs>=0.18.0" pytest numpy {pkg_name}"""
+        f"""$MAMBA_EXE create --yes --prefix {prefix} --platform=emscripten-32   python numpy "pyjs>=0.18.0" pytest  {pkg_name}"""
```

Comment:
does this matter?

Reply:
nah .. just debugging..removing and re-adding